src/test/unit/webview/utils/ErrorHandler.test.ts

```
@@ -331,19 +355,19 @@ describe('ErrorHandler', () => {
-      
+
       const report = errorHandler.formatErrorReport(error, {}, true);
-      
+
       expect(report).to.include('Error: Test error');
       expect(report).to.include('at test.js:1:1');
     });
 
     it('should handle errors without stack trace', () => {
       const error = new Error('Test error');
       delete error.stack;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)